services/crypto_delete.proto

```diff
@@ -29,18 +36,41 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Mark an account as deleted, moving all its current hbars to another account. It will remain in
- * the ledger, marked as deleted, until it expires. Transfers into it a deleted account fail. But a
- * deleted account can still have its expiration extended in the normal way.
+ * Delete an account.<br/>
+ * This will mark an account deleted, and transfer all tokens to a "sweep"
+ * account.
+ *
+ * A deleted account SHALL NOT hold a balance in any token type.<br/>
+ * A deleted account SHALL remain in state until it expires.<br/>
+ * Transfers that would increase the balance of a deleted account
+ * SHALL fail.<br/>
+ * A deleted account MAY be subject of a `cryptoUpdate` transaction to extend
+ * its expiration.<br/>
+ * When a deleted account expires it SHALL be removed entirely, and SHALL NOT
+ * be archived.
+ *
+ * ### Block Stream Effects
+ * None
  */
 message CryptoDeleteTransactionBody {
     /**
-     * The account ID which will receive all remaining hbars
+     * An account identifier.
+     * <p>
+     * The identified account SHALL receive all tokens from the deleted
+     * account.<br/>
+     * The identified account MUST sign this transaction.<br/>
+     * If not set, the account to be deleted MUST NOT have a balance in any
+     * token, a balance in HBAR, or hold any NFT.
      */
     AccountID transferAccountID = 1;
 
     /**
-     * The account ID which should be deleted
+     * An account identifier.
+     * <p>
+     * This account SHALL be deleted if this transaction succeeds.<br/>
+     * This account SHOULD not hold any balance other than HBAR.<br/>
```

Comment:
It may be possible for this line to be true, while the account is also designated as the treasury for a token type(s). I can't remember for sure if this would result in an `ACCOUNT_IS_TREASURY` error; it may be worth noting here.